COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp

```diff
@@ -335,6 +335,37 @@ namespace COLLADAMaya
             COLLADASW::URI sourceFileUri ( COLLADASW::URI::nativePathToUri ( currentScene ) );
             if ( sourceFileUri.getScheme ().empty () )
                 sourceFileUri.setScheme ( COLLADASW::URI::SCHEME_FILE );
+			if (sourceFileUri.scheme() == COLLADASW::URI::SCHEME_FILE && ExportOptions::sourceDataFormat() != ExportOptions::Absolute)
+			{
+				COLLADABU::URI relativeTo;
+				switch (ExportOptions::sourceDataFormat())
+				{
+					case ExportOptions::RelativeToExportedDAE:
+					{
+						relativeTo = COLLADABU::URI::nativePathToUri(getFilename());
+						break;
+					}
+					case ExportOptions::RelativeToMayaProjectWorkspace:
+					{
+						MString workspace;
+						if (MGlobal::executeCommand(MString("workspace -q -rd;"), workspace))
```

Comment:
Then base URI cannot be used since we want source_data path to be relative to Maya workspace and the other relative paths in the DAE to be relative to the DAE.
Can we use an <extra> in <COLLADA>?